.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -21,8 +24,7 @@ steps:
 
     steps:
       - label: ":ubuntu: Unit Tests"
-        command:
-          - ".buildkite/auditbeat/scripts/unit-tests.sh"
+        command: "mage -d ${BEATS_PROJECT_NAME} unitTest"
```

Comment:
while you can do `-d {project} ...` some commands internally assume that you are in the project dir and expect some files to be there. So explicitly cd-ing to the project dir for now should be done, just in case.

Reply:
To avoid cd-ing ES_BEAT env var should be used instead of PWD in [docker-compose.yml](https://github.com/elastic/beats/blob/04a73dcbc7a781bf8ddf49781879731b9fb8919b/filebeat/docker-compose.yml#L49). I have faced that issue in filebeat, meanwhile using "-d <beat-name>" worked successfully for integration tests in heartbeat.

Reply:
Yes I know, I don't want to increase the surface of the changes by changing the `docker-compose.yml`.